pyproject.toml

```diff
@@ -73,17 +73,11 @@ notebooks = [
   "ipysheet"
 ]
 
-[tools.setuptools]
-include-package-data = true
-
 [tool.setuptools.packages.find]
 where = ["."]
 include = ["pgfinder"]
 exclude = ["tests"]
 
-[tools.setuptools.package-data]
```

Comment:
This section configures the `package-data` that is to be included when building wheels and says that under the `pgfinder` package files with `.yaml` should be included.